hazelcast/src/main/java/com/hazelcast/sql/impl/client/SqlClientUtils.java

```diff
@@ -30,15 +31,28 @@ private SqlClientUtils() {
         // No-op.
     }
 
+    // Encode the exception as SqlError
     public static SqlError exceptionToClientError(Exception exception, UUID localMemberId) {
         HazelcastSqlException sqlException = CoreQueryUtils.toPublicException(exception, localMemberId);
 
+        boolean causeStackTraceExists = false;
+        String causeStackString = "";
+
+        // Get stack trace of the "cause" if it exists
+        Throwable cause = exception.getCause();
```

Comment:
Why isn't the top-level exception included?

Reply:
It did not want to duplicate the error message in the beginning. But may be will be helpful for debugging.
rev : 8418f58045b7b6924c31114a356300a37f3ca005